go/oasis-test-runner/scenario/e2e/basic.go

```diff
@@ -63,10 +63,23 @@ type basicImpl struct {
 	logger *logging.Logger
 }
 
+func (sc *basicImpl) Clone() scenario.Scenario {
+	return &basicImpl{
+		name:         sc.name,
+		clientBinary: sc.clientBinary,
+		clientArgs:   sc.clientArgs,
+		logger:       logging.GetLogger("scenario/e2e/" + sc.name),
```

Comment:
Why do we need to create a new logger that logs to the exactly same module as the previous one? You can share logger instances.